README.md

```diff
@@ -31,6 +31,97 @@ GetText/DecorateFunctionMessage:
   Enabled: true
 ```
 
+## Cops
+### GetText/DecorateFunctionMessage
+This cop looks for any raise / fail functions and checks the user visible message is using gettext decoration with the _() function.
+This cop will make sure the message is decorated as well as checking that the formatting of the message is compliant according to the follow rules.
+This cop supports autocorrecting of [Simple decoration of a message](#Simple-decoration-of-a-message). See the rubocop documentation on how to run autocorrect.
+
+#### Simple decoration of a message
+All simple message strings for should be decorated thusly.
+##### Error message thrown
+```
+'raise' function, message should be decorated
+```
+##### Bad
+``` ruby
+raise("Warning")
+```
+##### Good
+``` ruby
+raise(_("Warning"))
+```
+
+#### Multi-line message
+The message should not span multiple lines. This will cause issues when translating and with the gettext library.
+##### Error message thrown
+```
+'raise' function, message should not be a multi-line string
+```
+##### Bad
+``` ruby
+raise("this is a multi" \
+"line message")
+```
+##### Good
+``` ruby
+raise(_("this is a multi line message"))
+```
+
+#### Concatenated message
+The message should not concatenate multiple strings. This will cause issues in translation and with the gettext.
+##### Error message thrown
+```
+'raise' function, message should not be a concatenated string
+```
+##### Bad
+``` ruby
+raise("this is a concatenated" + "message")
+```
+##### Good
+``` ruby
+raise(_("this is a concatenated message"))
+```
+
+#### Interpolated message
+The message should be formated in a particular style. Otherwise this will cause issues in translation and with the gettext gem.
+##### Error message thrown
+```
+'raise' function, message should use correctly formatted interpolation
+```
+##### Bad
+``` ruby
+raise("this is an interpolated message IE #{variable}")
+```
+##### Good
+``` ruby
+raise(_("this is an interpolated message IE %{value0}") % {value0: var,})
+```
+
+#### No decoration
+The raise / fail function does not contain any decoration
+##### Error message thrown
+```
+'raise' function, should have decoration around the message
+```
+##### Bad
+``` ruby
+raise(someOtherFuntioncall(foo, "bar"))
+```
+##### Good
+Because the message does not contain any decoration or a simple message to decorate eg [Simple decoration of a message](#Simple-decoration-of-a-message) 
```

Comment:
I am not clear on what this sentence is saying; if there's no message to decorate, that seems clear enough, but it also says "because the message does not contain any decoration,"--but it seems like that's what this error message means, is that it doesn't have decoration when it should? Also isn't there supposed to be a message with a 'raise' function?

Anyway, I am confused by this section, so maybe you can explain it to me and I can help with this bit. :-) I think it's driving at "if there's no message that should be decorated" but I'm unclear.